vscode/vscode/vscode.mli

```diff
@@ -674,78 +582,73 @@ module ConfigurationTarget : sig
     | Workspace
     | WorkspaceFolder
 
-  val t_of_js : Ojs.t -> t
-
-  val t_to_js : t -> Ojs.t
+  include Js.T with type t := t
 end
 
 module WorkspaceConfiguration : sig
-  type t
-
-  type configurationTarget =
-    [ `ConfigurationTarget of ConfigurationTarget.t
-    | `Bool of bool
-    ]
+  include Js.T
 
-  type inspectResult =
+  type 'a inspectResult =
     { key : string
-    ; defaultValue : Jsonoo.t option
-    ; globalValue : Jsonoo.t option
-    ; workspaceValue : Jsonoo.t option
-    ; workspaceFolderValue : Jsonoo.t option
-    ; defaultLanguageValue : Jsonoo.t option
-    ; globalLanguageValue : Jsonoo.t option
-    ; workspaceLanguageValue : Jsonoo.t option
-    ; workspaceFolderLanguageValue : Jsonoo.t option
-    ; languageIds : string option
+    ; defaultValue : 'a option
+    ; globalValue : 'a option
+    ; workspaceValue : 'a option
+    ; workspaceFolderValue : 'a option
+    ; defaultLanguageValue : 'a option
+    ; globalLanguageValue : 'a option
+    ; workspaceLanguageValue : 'a option
+    ; workspaceFolderLanguageValue : 'a option
+    ; languageIds : string list option
     }
 
-  val get : t -> section:string -> unit -> Jsonoo.t option
+  val get : (module Js.T with type t = 'a) -> t -> section:string -> 'a option
```

Comment:
Is this ever called with anything other than `module Jsonoo`?

Reply:
I don't think so.  I'd like to leave this function as-is to be faithful to the VSCode API. 

I just added `get_json` and `inspect_json` functions for easier access to the JSON output.

Reply:
It's OK, the specializtion to json aren't really necessary. I was just thinking if `Ojs.t` should have been used instead. I suppose `get_default` becomes easier to use if we pass the conversion functions.